docs/using-the-compiler.rst

```diff
@@ -476,6 +477,17 @@ Output Description
               "legacyAssembly": {},
               // Bytecode and related details.
               "bytecode": {
+                // Debugging data at the level of functions.
+                "functionDebugData": {
+                  // Now follows a set of functions including compiler-internal and
+                  // user-defined function. The set does not have to be complete.
+                  "@mint_13": { // Internal name of the function
+                    "entryPoint": 128, // Byte offset into the code where the function starts
```

Comment:
I see that `entryPoint` can be omitted. This should say when it happens.

I see that it's when `m_tagPositionsInBytecode` contains `numeric_limits<size_t>::max()` for the tag, i.e. there was no code generated for the function? Is it the case of a function being optimized out/inlined?